x-pack/plugins/asset_inventory/common/types_api.ts

```diff
@@ -59,6 +88,16 @@ export interface EcsOrchestratorFieldset {
   };
 }
 
+export type CloudProviderName = 'aws' | 'gcp' | 'azure' | 'other' | 'unknown' | 'none';
```

Comment:
Out of the 3 other, unkown, none I would assume we only need 2. For example what is the difference between `other` cloud provider and `unkown`